### PR TITLE
notify only unblocked super users

### PR DESCRIPTION
**Pull request by eshiol**
### Summary of Changes
Fixed Privacy component sends GDPR requests to blocked super users

### Testing Instructions
Create a blocked super user
Create a GDPR request

### Expected result
The Privacy component sends GDPR requests only to unlocked users

### Actual result
Privacy component sends GDPR requests to blocked super users

### CODE REVIEW

**Comment by eshiol** [comment]
I propose to send notification only to super users with Receive System Emails = Yes
`->where($db->quoteName('sendEmail') . ' = 1')`

**Comment by zero-24** [comment]
@eshiol yes this makes sense please add it to this PR too.

File: administrator/components/com_messages/models/message.php [hunk 1]
```diff
@@ -465,7 +465,9 @@ public function notifySuperUsers($subject, $message, $fromUser = null)
 			$query = $db->getQuery(true)
 				->select($db->quoteName('user_id'))
 				->from($db->quoteName('#__user_usergroup_map'))
-				->where($db->quoteName('group_id') . ' IN(' . implode(',', $groups) . ')');
+				->where($db->quoteName('group_id') . ' IN(' . implode(',', $groups) . ')')
+				->where($db->quoteName('sendEmail') . ' = 1')
```
Comment (Quy) [inline administrator/components/com_messages/models/message.php]: Query has to join with `users` table to include these 2 fields in the `where` clause.

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on 5fe0f34afb9ef3b6a6b81e3c309f942da08efbf9<br /><br />works good to me thanks!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25829">issues.joomla.org/tracker/joomla-cms/25829</a>.</sub>

**Comment by viocassel** [comment]
I have tested this item :white_check_mark: successfully on 5fe0f34afb9ef3b6a6b81e3c309f942da08efbf9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25829">issues.joomla.org/tracker/joomla-cms/25829</a>.</sub>

**Comment by ghost** [comment]
Status "Ready To Commit".

**Comment by HLeithner** [comment]
Thank you for bringing GDPR requests to the right persons.

**Pull request merged**